docker/compose/public_full_node/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+# This compose file defines a Public Fullnode docker compose wrapper around libra-node. Currently
+# this can only connect to a local swarm instance, but work is in progress to enable more use
+# cases.
+#
+# In order to use, place a copy of the proper genesis.blob and waypoint.txt in this directory.
```

Comment:
Can these be discovered from a running network, and enforced if set?   Not for this pr, just a general observation.

Reply:
If we had engineered this concept in, unfortunately, we stumbled into this too late :/